status-im/status-react
# Implement collectible image preview

fixes #17326

### Summary

The lightbox screen is a screen for pictures preview.
Before it was used only for previewing sent photos in messages with images.
In the current PR it was untied from message data structures and moved to commons to be reused for previewing collectibles.
Previewing collectible image was made with this screen

![Simulator Screenshot - iPhone 13 - 2024-01-10 at 16 33 17](https://github.com/status-im/status-mobile/assets/5786310/663e385c-784a-465a-aaa0-d2ed547a84af)


### Testing notes
Impacted area is image preview in messages and collectible preview

### Testing steps
- Open Status
- Open wallet with collectibles
- Navigate to single collectible
- Press collectible image, make sure that image preview opened


status: ready
